client/styles/components/_overlay.scss

```diff
@@ -49,10 +56,13 @@
 
 /* Fixed height overlay */
 .overlay--is-fixed-height .overlay__body {
-  height: 80vh;
+  height: 100vh;
+
+  @media (min-width: 770px) {
+    height: 80vh;
+  }
```

Comment:
Now I'm playing with this and my head is exploding 🤯 but I think that `height: 100vh;` is actually all that we need?  Because the `max-height: 80%;` will apply over 651px so that's the same as `height: 80vh;`

Reply:
So I should remove the media query then?

Reply:
Yeah I think so.  I don't like to have code that doesn't do anything.  The overlay looks good though!

Reply:
done!